charts/rancher-thanos/v0.0.1/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: "0.9.0"
+description: A Helm chart for Rancher global monitoring
+name: rancher-thanos
+version: 0.0.1
+keywords:
+  - thanos
+  - prometheus
+  - metrics
+sources:
+  - https://github.com/thanos-io/thanos
+icon: https://raw.githubusercontent.com/thanos-io/thanos/master/docs/img/Thanos-logo_fullmedium.png
```

Comment:
Can you put the icon into the chart, please? Here is an example https://github.com/rancher/charts/blob/master/charts/grafana/v3.8.6/Chart.yaml#L8

So air gapped users can see the icon.